tests/browsercontext-add-cookies.spec.ts

```diff
@@ -206,7 +206,8 @@ it('should set cookie with reasonable defaults', async ({context, server}) => {
   await context.addCookies([{
     url: server.EMPTY_PAGE,
     name: 'defaults',
-    value: '123456'
+    value: '123456',
+    sameSite: 'Lax',
```

Comment:
In this particular test we check "reasonable defaults", so I'd rather have different expectations per browser.

Reply:
Done